app/setup.py

```diff
@@ -59,7 +59,7 @@
         'https://fonts.googleapis.com',
         "'unsafe-inline'",
     ],
-    'connect-src': ["'self'", 'https://cdn.ons.gov.uk'],
+    'connect-src': ["'self'", 'ws://localhost:35729', 'https://cdn.ons.gov.uk'],
```

Comment:
Is this for live reload? We probably should only be enabling it through a configuration as per setup_secure_headers does with the EQ_ENABLE_LIVE_RELOAD flag?

Reply:
Yeah it's for live reload... Could you give me a hand with that?

Reply:
Think you should just be able to add it here, given the flag already exists: https://github.com/ONSdigital/eq-survey-runner/blob/0552d46048dcd77deff59b4f833a03d1cf39821f/app/setup.py#L236

Reply:
Cheers, updated this now